backend/baseApp/env.ts

```diff
@@ -0,0 +1,11 @@
+import * as env from 'env-var'
```

Comment:
Should we standardize on using `env-var` — I think core uses it as well. If so then I can switch to this instead of the `coreContext` loader I copied from engage backend.

Reply:
Great one :) I use it in some areas as well and it offers more features I think than the `coreContext` loader. Maybe we can use take lightweight package for now 🤔